### PR TITLE
Added external references for data views

### PR DESCRIPTION
**Pull request by stanasse**
Created 6 yaml files for external references to implement Data Views documentation generation using docupipe.

The external references are being used to generate documentation using docupipe and exist in the [internal ](https://dev.azure.com/osieng/engineering/_wiki/wikis/OCS%20Internal%20API%20Documentation?friendlyName=data-views&pagePath=%2FIntegration&wikiVersion=GBwiki)and [sandbox ](https://github.com/osisoft/APIDocumentationSandbox/tree/Integration/Content_Portal/Documentation/Integration)documentation sites.

The existing data views public documentation is here.

### CODE REVIEW

**Comment by stanasse** [comment]
The corresponding PR for DataViews that uses the external references is [PR 21469](https://dev.azure.com/osieng/engineering/_git/ocs-data-views/pullrequest/21469)

File: content/external-references/dataviews-summaries.yaml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+dataviews-acl-update:
+  type: markdown
+  value: |
+    Update the default [`AccessControlList`](https://ocs-docs.osisoft.com/Content_Portal/Documentation/Access_Control.html#access-control-lists) for the DataViews collection.
+
+# method summaries: requesting data
+data-get:
+  type: markdown
+  value: |
+    Get data for the provided index parameters with paging. See [documentation on paging](https://ocs-docs.osisoft.com/Content_Portal/Documentation/DataViews/GetDataViewData/Quick_Start_Get_Data_View_Data.html#paging) for further information.
```
Comment (GiselleLeeOSI) [inline content/external-references/dataviews-summaries.yaml:62]: We're hoping to use present tense at summary level, so in line 62: Gets data.

File: content/external-references/dataviews-summaries.yaml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+dataviews-acl-get:
+  type: markdown
+  value: |
+    Get the default [`AccessControlList`](https://ocs-docs.osisoft.com/Content_Portal/Documentation/Access_Control.html#access-control-lists) for the DataViews collection.
+
+dataviews-acl-update:
+  type: markdown
+  value: |
+    Update the default [`AccessControlList`](https://ocs-docs.osisoft.com/Content_Portal/Documentation/Access_Control.html#access-control-lists) for the DataViews collection.
```
Comment (GiselleLeeOSI) [inline content/external-references/dataviews-summaries.yaml:56]: We're hoping to use present tense at summary level, so in line 51: Gets the default. Same for all others here.

**Review by GiselleLeeOSI — APPROVED** [review]
Looks good. Thank you for putting it together, Steve!

File: content/external-references/dataviews-summaries.yaml (hunk 1 above)
Comment (stanasse) [reply content/external-references/dataviews-summaries.yaml]: Revised.

File: content/external-references/dataviews-summaries.yaml (hunk 2 above)
Comment (stanasse) [reply content/external-references/dataviews-summaries.yaml]: Revised.

File: content/external-references/dataviews-summaries.yaml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+dataview-get:
+  type: text/plain
+  value: |
+    Returns the specified data view.
+
+dataview-getorcreate:
+  type: markdown
+  value: |
+    This call creates the specified data view. If a data view with the same id already exists, the existing data view is compared with the specified data view. If they are identical, a redirect (`302 Found`) is returned with the `Location` response header indicating the URL where the data view may be retrieved using a Get function. If the data views do not match, the request fails with `409 Conflict`.
```
[the remark this answers is not in the document]
Comment (stanasse) [reply content/external-references/dataviews-summaries.yaml]: Revised.

**Comment by GiselleLeeOSI** [comment]
Looks very good. Thank you. As mentioned earlier, we'll need to update the links to the new documentation site at a later time.

**Pull request merged**